lib/setopt.c

```diff
@@ -654,7 +654,7 @@ CURLcode Curl_vsetopt(struct Curl_easy *data, CURLoption option, va_list param)
     data->set.upload = FALSE;
     break;
 
-#ifndef CURL_DISABLE_MIME
+#ifndef CURL_DISABLE_FORM_API
```

Comment:
Doesn't disabling MIME also imply disabling the form API? This makes them look like they are independent from each other, but surely the form API doesn't work if you disable MIME does it?

Reply:
> Doesn't disabling MIME also imply disabling the form API?

Yes. The case is tested in `configure.ac` and `CURL_DISABLE_FORM_API` is always set when `CURL_DISABLE_MIME` is. `CMakeLists.txt` doesn't check it though.

Reply:
This now also handled in CMake.